Results of doctest builder run on 2014-09-28 16:59:13
=====================================================

Document: index
---------------
**********************************************************************
File "index.rst", line 6, in default
Failed example:
    main = fb.Surface()
Expected nothing
Got:
    surpress debug info
**********************************************************************
File "index.rst", line 10, in default
Failed example:
    sub.blit("../examples/cylon.png")
Expected:
    0
Got:
    -1
**********************************************************************
File "index.rst", line 10, in default
Failed example:
    main.clear()
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.7/doctest.py", line 1289, in __run
        compileflags, 1) in test.globs
      File "<doctest default[3]>", line 1, in <module>
        main.clear()
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 732, in clear
        self.informdriver()
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 637, in informdriver
        a=fblib.fbsetwinparams( self.origo[0], self.origo[1], self.size[0], self.size[1],
    TypeError: 'NoneType' object has no attribute '__getitem__'
**********************************************************************
File "index.rst", line 28, in default
Failed example:
    sub.addpoly(x1,y1,z1)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.7/doctest.py", line 1289, in __run
        compileflags, 1) in test.globs
      File "<doctest default[12]>", line 1, in <module>
        sub.addpoly(x1,y1,z1)
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 119, in wrapper
        return self(instance, *args, **kwargs)
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 115, in __call__
        return self.function(*args, **kwargs)
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 904, in addpoly
        return fblib.fbaddpoly(x1,y1,z1)
      File "fblib.pyx", line 325, in fbpy.utils.fblib.fbaddpoly (./fbpy/utils/fblib.c:4146)
    TypeError: fbaddpoly() takes exactly 4 positional arguments (3 given)
**********************************************************************
File "index.rst", line 30, in default
Failed example:
    sub.addpoly(x2,y2,z2)
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.7/doctest.py", line 1289, in __run
        compileflags, 1) in test.globs
      File "<doctest default[13]>", line 1, in <module>
        sub.addpoly(x2,y2,z2)
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 119, in wrapper
        return self(instance, *args, **kwargs)
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 115, in __call__
        return self.function(*args, **kwargs)
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 904, in addpoly
        return fblib.fbaddpoly(x1,y1,z1)
      File "fblib.pyx", line 325, in fbpy.utils.fblib.fbaddpoly (./fbpy/utils/fblib.c:4146)
    TypeError: fbaddpoly() takes exactly 4 positional arguments (3 given)
**********************************************************************
File "index.rst", line 32, in default
Failed example:
    sub.drawpolys()
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.7/doctest.py", line 1289, in __run
        compileflags, 1) in test.globs
      File "<doctest default[14]>", line 1, in <module>
        sub.drawpolys()
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 959, in drawpolys
        return fblib.fbdrawpolys()
    TypeError: fbdrawpolys() takes exactly one argument (0 given)
**********************************************************************
File "index.rst", line 34, in default
Failed example:
    sub.trafo.rotate(np.pi/2)
Expected:
    0
Got nothing
**********************************************************************
File "index.rst", line 36, in default
Failed example:
    sub.drawpolys()
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.7/doctest.py", line 1289, in __run
        compileflags, 1) in test.globs
      File "<doctest default[16]>", line 1, in <module>
        sub.drawpolys()
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 959, in drawpolys
        return fblib.fbdrawpolys()
    TypeError: fbdrawpolys() takes exactly one argument (0 given)
**********************************************************************
File "index.rst", line 12, in default
Failed example:
    main.informdriver()
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.7/doctest.py", line 1289, in __run
        compileflags, 1) in test.globs
      File "<doctest default[2]>", line 1, in <module>
        main.informdriver()
      File "/usr/local/lib/python2.7/dist-packages/fbpy/fb.py", line 637, in informdriver
        a=fblib.fbsetwinparams( self.origo[0], self.origo[1], self.size[0], self.size[1],
    TypeError: 'NoneType' object has no attribute '__getitem__'
**********************************************************************
File "index.rst", line 16, in default
Failed example:
    sub.printxy((10,10),"Hello world!", 2)
Expected:
    0 
Got:
    0
**********************************************************************
File "index.rst", line 30, in default
Failed example:
    sub.grabsilent("./source/images/printxy.png")
Expected:
    0               
Got:
    0
1 items passed all tests:
   1 tests in >>> import fbpy.fb
**********************************************************************
1 items had failures:
  11 of  88 in default
89 tests in 2 items.
78 passed and 11 failed.
***Test Failed*** 11 failures.

Doctest summary
===============
   89 tests
   11 failures in tests
    0 failures in setup code
    0 failures in cleanup code
